client/js/test/table-view.test.js

```diff
@@ -80,6 +80,20 @@ describe('table-view', () => {
 			const trs = document.querySelectorAll('TBODY TR');
 			expect(trs[1].cells[2].textContent).toBe('123');
```

Comment:
Add at least 3-4 additional tests for your new feature. You should be writing tests as you develop, so that you can quickly catch bugs and broken mental models.